Src/IronPython/Compiler/Parser.cs

```diff
@@ -2710,31 +2742,51 @@ private Parameter ParseDefParameter(HashSet<string> names, ParameterKind paramet
             return ret;
         }
 
-        // listmaker: expression ( list_for | (',' expression)* [','] )
+        // testlist_comp: (test|star_expr) ( comp_for | (',' (test|star_expr))* [','] )
         private Expression FinishListValue() {
+
             var oStart = GetStart();
             var oEnd = GetEnd();
-            int grouping = _tokenizer.GroupingLevel;
+            var grouping = _tokenizer.GroupingLevel;
 
             Expression ret;
             if (MaybeEat(TokenKind.RightBracket)) {
                 ret = new ListExpression();
-            } else {
-                bool prevAllow = _allowIncomplete;
+            }
```

Comment:
Just a note on style: `} else {`

If you're using VS then formatting the document should pick up the editorconfig and apply the correct style throughout.

Reply:
I've got VS and also ReSharper. My auto-format is currently doing too much as it is setup to match the StyleCop format, however closing an outer brace works.

I could take a look at formalizing the coding standard with some analyzers if you are interested?

Reply:
Is there an analyzer that picks up the rules from editorconfig? Some parts of the codebase (e.g. BZip2 and c#sqlite) have their own inconsistent style (although I haven't set up an editorconfig for those).

Reply:
No, but VS with ReSharper should pick it up.